IBAnimatable/BlurDesignable.swift

```diff
@@ -28,11 +28,14 @@ public extension BlurDesignable where Self: UIView {
    configureBlurEffectStyle method, should be called in layoutSubviews() method
    */
   public func configureBlurEffectStyle() {
+    configureBlurEffectStyle(self)
+  }
+  public func configureBlurEffectStyle(_ blurableView: UIView) {
```

Comment:
shall we use `public func configureBlurEffectStyle(for blurableView: UIView)`? using `for` instead of `_` there.